File: maths/fizz_buzz.py
#Given an integer n, return a string array answer (1-indexed) where:

# answer[i] == "FizzBuzz" if i is divisible by 3 and 5.
# answer[i] == "Fizz" if i is divisible by 3.
# answer[i] == "Buzz" if i is divisible by 5.
# answer[i] == i (as a string) if none of the above conditions are true.

def fizz_buzz(n):
    result = []

    for i in range(1,n+1):
        strn = ''
        if i % 3 == 0:
            strn += 'Fizz'
        if i % 5 == 0:
            strn += 'Buzz'
        if not strn:
            strn += str(i)
        result.append(strn)
    
    return result

File: math/fizzBuzz.js
// Given an integer n, return a string array answer (1-indexed) where:

// answer[i] == "FizzBuzz" if i is divisible by 3 and 5.
// answer[i] == "Fizz" if i is divisible by 3.
// answer[i] == "Buzz" if i is divisible by 5.
// answer[i] == i (as a string) if none of the above conditions are true.

/**
 * @param {number} n
 * @return {string[]}
 */
var fizzBuzz = function (n) {
  let result = [];

  for (let i = 1; i <= n; i++) {
    let string = "";
    if (i % 3 === 0) string += "Fizz";
    if (i % 5 === 0) string += "Buzz";

    if (string === "") string += i;
    result.push(string);
  }

  return result;
};

